### DISCUSSION TITLE
PIR sensor to turn on lights delay

### DISCUSSION THREAD

**Discussion by HallCraig** [rainepretorius/olarm-ha-integration#84] [General]
Hi 

I'm trying to figure out if at all possible to make the PIR sensor a bit quicker to auto-switch on lights once motion is detected.

I know it may be API limits/default settings. Any way to adjust - all I'm, wanting for now is to integrate with the PIR motion sensors once activated. 

Much appreciated

**Comment by rainepretorius** [comment]
It is not possible as Olarm has the limiter ant it takes a few seconds or minites before the state changes on their APi. It is immediate on their app, so it is not a device issue and thus they must be doing this intentionally.
